models/spring-ai-ollama/src/test/java/org/springframework/ai/ollama/OllamaChatClientIT.java

```diff
@@ -63,7 +63,7 @@ class OllamaChatClientIT {
 	private static final Log logger = LogFactory.getLog(OllamaChatClientIT.class);
 
 	@Container
-	static GenericContainer<?> ollamaContainer = new GenericContainer<>("ollama/ollama:0.1.29").withExposedPorts(11434);
+	static OllamaContainer ollamaContainer = new OllamaContainer("ollama/ollama:0.1.32").withExposedPorts(11434);
```

Comment:
```suggestion
	static OllamaContainer ollamaContainer = new OllamaContainer("ollama/ollama:0.1.32");
```

Reply:
Good catch, thanks @eddumelendez! I have committed the change to remove the unnecessary exposed port setting.